Symphony/Classes/LPMessage.m

```diff
@@ -10,4 +10,29 @@
 
 @implementation LPMessage
 
+- (NSDictionary *) getClassMapping {
+    NSDictionary *mapping = @{@"id": @"messageId",
```

Comment:
evne though `id` represents an `NSObject`, you can actually use id as an instance variable name as well